tools /kafka/README.md

```diff
@@ -0,0 +1,46 @@
+
+# Kafka manager with web UI to monitor and maintain Kafka's clusters
+
+## General Information
+The container on: [Docker Redpanda Console](https://hub.docker.com/r/redpandadata/console)
+
+You can find the documentation of the service here: [Redpanda Console repository](https://github.com/redpanda-data/console)
+
+## Deployment
+The service requests at least 50 mCPU and 500 megabytes of RAM.
```

Comment:
I never tested how much exactly RAM I need.
Now I see the 50MB in use, but it may need more depending on the task.
![image](https://github.com/thingsboard/thingsboard-ce-k8s/assets/79898499/9415ce93-2059-40f1-bc0e-d899fd6c9199)

Reply:
Changed the description in the Readme and the deployment file to save resources.